src/platforms/javascript/common/sourcemaps/tools/webpack.mdx

```diff
@@ -77,16 +77,7 @@ Set up the `SentryWebpackPlugin` as the last running plugin, otherwise, the resu
 
 If you prefer to upload source maps manually, configure Webpack to output source maps:
 
-```javascript
-module.exports = {
-  output: {
-    path: path.join(__dirname, "dist"),
-    filename: "[name].js",
-    sourceMapFilename: "[name].js.map",
-  },
-  // other configuration
-};
-```
+<PlatformContent includePath="configuration/sourcemaps/tools/webpack/advanced-usage" />
```

Comment:
It's occurring to me: what we're showing them _doesn't_ actually "configure webpack to output sourcemaps." It configures the name of the sourcemaps, but it doesn't actually make the sourcemaps happen. Further, though the naming scheme is important (so that sentry-cli will auto-detect the maps), the output location isn't.

Reply:
Wdyt about "If you prefer to upload source maps manually, configure Webpack to set appropriate names", before the platform content?

Reply:
Sorry - I should have been clearer. I think the wording is fine. That comment was motivation for the code changes I suggested, so that now the code _does_ match the existing words.